django/contrib/admin/options.py

```diff
@@ -708,7 +708,8 @@ def log_addition(self, request, object):
             content_type_id=get_content_type_for_model(object).pk,
             object_id=object.pk,
             object_repr=force_text(object),
-            action_flag=ADDITION
+            action_flag=ADDITION,
+            change_message=message
```

Comment:
That would make this inconsistant with https://github.com/django/django/pull/4335/files#diff-3c42de3e53aba87b32c494f995a728dfR728 and https://github.com/django/django/pull/4335/files#diff-3c42de3e53aba87b32c494f995a728dfR744. Shall I add a trailing comma on these lines as well?